cmd/cpe2cve/cpe2cve.go

```diff
@@ -176,6 +176,8 @@ func processInput(in io.Reader, out io.Writer, caches map[string]*cvefeed.Cache,
 }
 
 func init() {
+	flog.AddFlags(flag.CommandLine, nil)
+	stats.AddFlags()
```

Comment:
I'd rather do it in `Main`: it would be consistent with `cfg.addFlags()` call that's already there and init() also pollutes the tests, which is not a big deal, of course, but something to keep in eye at.